lib/gscraper_web/views/shared_view.ex

```diff
@@ -0,0 +1,21 @@
+defmodule GscraperWeb.SharedView do
+  use GscraperWeb, :view
+
+  alias Gscraper.Guardian.Authentication
+
+  def current_user(conn), do: Authentication.get_current_user(conn)
```

Comment:
Not sure should we set current user to conn or move this to a helper instead 🤔  I think eventually we will need to get current user on the controller as well 💭

Reply:
<img width="919" alt="Screen Shot 2021-09-26 at 14 56 36" src="https://user-images.githubusercontent.com/1896814/134799011-24f776d1-9311-4ff2-8fcf-58716f8a2fac.png">

In fact, the current user is set into the `conn` already. As in the above screenshot, when inspecting the conn, it shows that the current user exists in the conn. It was handled automatically by the `Guardian` package. The `Gscraper.Guardian.Authentication` is an implementation of that package and it also acts as a helper in case we need to get the current user as in the code above.